From 419238eada6257781c49fedc95dd57208433afe2 Mon Sep 17 00:00:00 2001
From: Ma Wupeng <mawupeng1@huawei.com>
Date: Wed, 31 Aug 2022 09:50:27 +0800
Subject: [PATCH 057/261] [Huawei] mm: mem_reliable: Make some functions static

Offering: HULK
hulk inclusion
category: feature
bugzilla: 187270

--------------------------------

Add static to memory reliable related functions

Signed-off-by: Ma Wupeng <mawupeng1@huawei.com>
---
 mm/mem_reliable.c | 20 ++++++++++++--------
 1 file changed, 12 insertions(+), 8 deletions(-)

diff --git a/mm/mem_reliable.c b/mm/mem_reliable.c
index 4540cfc76489..28baba72bb5e 100644
--- a/mm/mem_reliable.c
+++ b/mm/mem_reliable.c
@@ -204,8 +204,9 @@ void reliable_report_meminfo(struct seq_file *m)
 }
 
 #ifdef CONFIG_SYSCTL
-int reliable_limit_handler(struct ctl_table *table, int write,
-	void __user *buffer, size_t *length, loff_t *ppos)
+static int reliable_limit_handler(struct ctl_table *table, int write,
+				  void __user *buffer, size_t *length,
+				  loff_t *ppos)
 {
 	unsigned long old = task_reliable_limit;
 	int ret;
@@ -223,8 +224,9 @@ int reliable_limit_handler(struct ctl_table *table, int write,
 	return ret;
 }
 
-int reliable_pagecache_max_bytes_write(struct ctl_table *table, int write,
-	void __user *buffer, size_t *length, loff_t *ppos)
+static int reliable_pagecache_max_bytes_write(struct ctl_table *table,
+					      int write, void __user *buffer,
+					      size_t *length, loff_t *ppos)
 {
 	unsigned long old_value = reliable_pagecache_max_bytes;
 	int ret;
@@ -283,8 +285,9 @@ static void mem_reliable_disable_all(void)
 	pr_info("memory reliable feature disabled.\n");
 }
 
-int reliable_debug_handler(struct ctl_table *table, int write,
-	void __user *buffer, size_t *length, loff_t *ppos)
+static int reliable_debug_handler(struct ctl_table *table, int write,
+				  void __user *buffer, size_t *length,
+				  loff_t *ppos)
 {
 	unsigned long old_ctrl_bits, new_ctrl_bits;
 	static DEFINE_MUTEX(reliable_debug_mutex);
@@ -318,8 +321,9 @@ int reliable_debug_handler(struct ctl_table *table, int write,
 #ifdef CONFIG_SHMEM
 static unsigned long sysctl_shmem_reliable_bytes_limit = ULONG_MAX;
 
-int reliable_shmem_bytes_limit_handler(struct ctl_table *table, int write,
-		void __user *buffer, size_t *length, loff_t *ppos)
+static int reliable_shmem_bytes_limit_handler(struct ctl_table *table,
+					      int write, void __user *buffer,
+					      size_t *length, loff_t *ppos)
 {
 	unsigned long *data_ptr = (unsigned long *)(table->data);
 	unsigned long old = *data_ptr;
-- 
2.17.1

